Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -404,14 +408,26 @@ def set(self, issue, assignee=None, opened=None, why=None, project=None, compone
                 error_message="Failed to modify '{}'".format(issue)
             )
             if not response:
-                if assignee:
-                    issue._assignee = None
                 if opened is not None:
                     issue._opened = None
                 return None
 
         return self.add_comment(issue, why) if why else issue
 
+    def add_assignees(self, issue, assignees):
+        response = self.request(
+            'issues/{id}/assignees'.format(id=issue.id),
+            method='POST',
+            authenticated=True,
+            json={'assignees': assignees},
+            error_message="Failed to modify '{}'".format(issue)
+        )
+        if not response:
+            issue._assignee = None
+            return False
```

Comment:
Should return an empty list, I think, to be consistent.

Reply:
The `all()` function returns True if everything in the generator is True, otherwise it returns False. I am considering switching it to return the list of assignees if that would make more sense here.